### PR TITLE
Woop woop - Alarm work

### PR DESCRIPTION
**Pull request by jusvit**
Added model and textures(On texture included but not implemented yet),
boundryboxes, name localization and translations.

TODO: On/off variants, Placeable on walls/roofs(faces), FIX IT so it's visible when holding in hand(Model is out of screen), and finally add a recipe for it.

I am unsure of how to go about the remaining stuff so help would be appreciated.
capture

### CODE REVIEW

File: src/main/resources/assets/techreborn/lang/fr_fr.lang [hunk 1]
```diff
@@ -898,3 +898,6 @@ achievement.TechReborn:ore_Pickup=Nouveau minerai !
 achievement.TechReborn:ore_Pickup.desc=Minez un minerai techreborn
 achievement.TechReborn:centrifuge_Craft=Il est temps de tourner
 achievement.TechReborn:thermalgen_Craft=Il est temps de chauffer
+
+#Blocks
+tile.techreborn:alarm.name=Alarme
```
Comment (modmuss50) [inline src/main/resources/assets/techreborn/lang/fr_fr.lang:903]: Im not sure how you go theses, just checking if you did use google translate or something dont.

**Review by modmuss50 — CHANGES_REQUESTED** [review]
PNG images should be 16x16 not 464!

File: src/main/resources/assets/techreborn/lang/fr_fr.lang (hunk 1 above)
Comment (jusvit) [reply src/main/resources/assets/techreborn/lang/fr_fr.lang:903]: I literally typed alarm to each language and confirmed with other sources besides  Google translate, they should prove a valid placeholder till someone can confirm.

Comment (jusvit) [reply src/main/resources/assets/techreborn/lang/fr_fr.lang:903]: Such a simple word 'Alarm', it's pretty easy to find the proper name in other languages. As alarm is generally a universally used word.

**Comment by jusvit** [comment]
Textures has been resized

**Comment by modmuss50** [comment]
Ok we can go with it then, if it’s wrong I’m sure people will open a pull request to fix it.

**Comment by jusvit** [comment]
@modmuss50  This pull request is only to put what I have done on the alarm side branch, it isn't release ready.

Also I've fulfilled your requests, a merge is called upon.

**Pull request merged**